src/components/Mission/MissionPage.js

```diff
@@ -30,6 +31,27 @@ const Missions = () => {
               <tr key={data.mission_id}>
                 <td className="Title">{data.mission_name}</td>
                 <td>{data.description}</td>
+                <td>
+                  {data.reserved && (
+                    <span style={{ background: 'green' }}>Active member</span>
+                  )}
+                  {data.reserved && <span>NOT A MEMBER </span>}
+                </td>
+                <td className="updateBtn">
+                  <button
+                    type="button"
+                    onClick={() => dispatch(reservationUpdates(data.mission_id))}
+                    style={{
+                      border: !data.reserved
+                        ? '1px solid black'
+                        : '1px solid red',
+                      backgroundColor: 'white',
+                      color: !data.reserved ? 'black' : 'red',
+                    }}
+                  >
+                    {!data.reserved ? 'Join mission' : 'Leave mission'}
+                  </button>
+                </td>
```

Comment:
[OPTIONAL] One of the essences of the component is for simplicity of pages, it would be fine to keep the HTML in another file which is not the page file itself. Then import it as JSX.

Reply:
Alright I will work on that, thanks for the review @Olamarx